src/engine/layout.jl

```diff
@@ -172,7 +172,21 @@ function tex_layout(expr, state)
                     (rightinkbound(content), 0)
                 ]
             )
+        elseif head == :overline
+            content = tex_layout(args[1], state)
+
+            lw = thickness(font_family)
+            y =  topinkbound(content) - lw
+
+            hline = HLine(inkwidth(content), lw)
 
+            return Group(
+                [hline, content],
+                Point2f[
+                    (0, y + lw/2),
```

Comment:
Currently it looks like a bit too tight I believe. Adding a bit of padding here (e.g. `y + lw/2 + 0.1`, but the exact value need to be tested) should solve it.

![image](https://user-images.githubusercontent.com/17861033/192152152-12a5a7c0-c13d-484f-8f2f-c8273f0040f5.png)

I generate the image using the file in `prototype/` (need to dev `Makie`, `MakieCore` and `CairoMakie` for it to work currently I think).

Reply:
I adjusted the height and length to match those of the \bar command.

The following image is created by the LaTeX String `L"\bar{n} \overline{n} \overline{A + B} \sqrt{A + B}"`
![test](https://user-images.githubusercontent.com/64231/192824102-54d0e027-2d5e-4a0d-8eb1-ac043156a8c4.png)